.gitignore

```diff
@@ -6,7 +6,8 @@ logs
 node_modules
 
 # Generated files
-dist
```

Comment:
Please add this back and remove generated files from the repository. They will be generated and put on npm during the build phase.